Configurations/make-release-package.sh

```diff
@@ -115,7 +115,7 @@ if [ "$ACTION" = "" ] ; then
     # Generate zip containing the xcframework for SPM
     cd "$CONFIGURATION_BUILD_DIR/staging-spm"
     #rm -rf "$CONFIGURATION_BUILD_DIR/Sparkle.xcarchive"
-    zip -rqyX -9 "../Sparkle-for-Swift-Package-Manager.zip" *
+    ditto -c -k --zlibCompressionLevel 9 --rsrc . "../Sparkle-for-Swift-Package-Manager.zip"
     
     # GitHub actions set the CI environment variable to true
     if [ "$CI" != true ]; then
```

Comment:
When we are doing this specific workflow in CI, we will need to use the marketing version as the tag name. So Package.swift will be updated, committed, and pushed appropriately.

Reply:
Everything in this make-release-package.sh script needs to use `MARKETING_VERSION` now, not `CURRENT_PROJECT_VERSION`

Reply:
Done in e2cd36410d0c3d5357c0128515f0ae19e8e54e39.

Reply:
I've also removed the CI check here, as you suggested: 13df5a238eb1049cec4be7b59d3ded19b119d79f